### ISSUE TITLE
logging into MPD in browser not supported

### ISSUE THREAD

**Issue by auroraanna** [vijfhoek/empede#1]
The app doesn't seem to support specifying the password for MPD (if there is one). I can't use this client because MPD is available to my entire local network, so I need to set a password.

**Comment by vijfhoek** [comment]
Good call. Would being able to set it using an environment variable work for you?

**Comment by vijfhoek** [comment]
I've added the `MPD_PASSWORD` environment variable in the latest release. I'll be leaving this issue open though, since I'd like to allow logging in in the browser in the future

**Comment by auroraanna** [comment]
Should I rename this accordingly?

**Comment by vijfhoek** [comment]
Sure
